src/main/java/uk/gov/ons/census/fwmtadapter/services/CaseService.java

```diff
@@ -0,0 +1,36 @@
+package uk.gov.ons.census.fwmtadapter.services;
+
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Service;
+import org.springframework.util.StringUtils;
+import org.springframework.web.client.RestTemplate;
+import uk.gov.ons.census.fwmtadapter.model.dto.CaseIdDto;
+
+@Service
+public class CaseService {
```

Comment:
This is not a service... it's a client. I suggest it should live in a package called `client` and it should probably be annotated `Component` instead of `Service`.

Reply:
oh and it should be called CaseApiClent or something like that.